sentinelhub/constants.py

```diff
@@ -296,11 +296,8 @@ class MimeType(Enum):
     XML, GML, RAW
     """
     TIFF = 'tiff'
-    TIFF_d8 = 'tiff;depth=8'
-    TIFF_d16 = 'tiff;depth=16'  # This is the same as TIFF
-    TIFF_d32f = 'tiff;depth=32f'
     PNG = 'png'
-    JPG = 'jpg'
+    JPG = 'jpeg'
```

Comment:
Because according to [SH reference](https://docs.sentinel-hub.com/api/latest/reference/#operation/process) (see output -> format) it is supposed to be `image/jpeg`. 

At the end of the day, requesting either `image/jpg` and `image/jpeg` works, so if you prefer, I can revert.

Reply:
Ok, thanks. It seems that `jpeg` might actually be more correct, let's keep it this way.